src/Microsoft.ML.Data/Model/Onnx/OnnxContext.cs

```diff
@@ -7,6 +7,8 @@
 
 namespace Microsoft.ML.Runtime.Model.Onnx
 {
+    public enum OnnxVersion { Latest, Experimental }
```

Comment:
>public enum OnnxVersion { Latest, Experimental  [](start = 4, length = 47)

This doesn't feel right.
First of all, I would expect not enum, but some kind of string.
Second why Latest version is first in list?
Third, you need to assign values.

Reply:
Hello,
(1) I was using string but then agreed with @TomFinley that string is ambigous and the number of supported versions is likely tiny.
(2) Maybe change Latest to Stable could be better? I assume that the first option is the option a user may prefer.
(3) What do you mean by assign value? It's an enum declaration.
Thanks,


---
In reply to: [219627520](https://github.com/dotnet/machinelearning/pull/947#discussion_r219627520) [](ancestors = 219627520)

Reply:
3) public enum OnnxVersion: byte { Stable=0, Experimental=1} 

---
In reply to: [219651774](https://github.com/dotnet/machinelearning/pull/947#discussion_r219651774) [](ancestors = 219651774,219627520)

Reply:
Done!